### ISSUE TITLE
The `Add Another Item` Button on multiple value field form is too specific

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#5235]
**Description of the bug**

I often use fields to describe things that are not items, so having the button read `Add another item` seems very strange. In today's example I am using this on a field for "Authors".

I would like to prose we change the button text to `Add another` which is likely to work for all fields, rather than only those containing items.

**Comment by jenlampton** [comment]
I've filed [a PR which changes the button label](https://github.com/backdrop/backdrop/pull/3746). Adding screenshots to the Original Post.

**Comment by klonos** [comment]
Wouldn't it be more useful if we allowed the label of the button to be customized if the "Number of values" under the "Global settings" fields set for the field is set to either something other than 1 or unlimited?

There's a tiny contrib that already does that: https://www.drupal.org/project/custom_add_another (also allows customizing the label of the "Remove" button). Perhaps we should port that to contrib?

**Comment by jenlampton** [comment]
I was just going to say "yes, it would be useful, but that's a job for contrib" but you beat me to it! In this issue, I am trying to make core seem less broken :)

**Comment by klonos** [comment]
Yes, if you see it that way, then the generic "Add another" may be better than "Add another item".

Many tests are failing in the PR.

**Comment by jenlampton** [comment]
Updated the PR to fix failing tests.

**Comment by klonos** [comment]
Looks good, and works as expected 👍🏼

**Comment by BWPanda** [comment]
Maybe I've been away from Backdrop too long (or maybe I'm just tired and not thinking straight), but how was the screenshot made in core? To me that looks like Paragraphs...

**Comment by indigoxela** [comment]
> how was the screenshot made in core? To me that looks like Paragraphs...

Could be Multifield...

Here are screenshots for comparison - with a simple textfield:

![add-another-item](https://user-images.githubusercontent.com/842066/133882057-9579181b-95e1-4a7f-a173-153d548bd568.png)

![add-another](https://user-images.githubusercontent.com/842066/133882063-c3ecaf13-739b-410c-955c-618613d65578.png)

Both seem OK to me, but I get the point that "item" may or may not fit the field. The neutral "Add another" might be the better option.

**Comment by olafgrabienski** [comment]
Related: #3520 (Maybe a good moment to also finish that one.)

**Comment by jenlampton** [comment]
The screenshot shows a paragraphs field, yes, but the button (this issue)
is from core :)

**Comment by BWPanda** [comment]
> The screenshot shows a paragraphs field, yes, but the button (this issue)
is from core :)

Ah, interesting...

**Comment by quicksketch** [comment]
With the small breakages seen in the tests, I'd feel more comfortable about this being in a minor release rather than a bug fix.

It doesn't seem like there's any strong opposition here, but also not a real strong approval. Still seems RTBC to me.
